recipe-server/client/control_new/components/recipes/RecipeDetailPage.js

```diff
@@ -48,8 +49,15 @@ export default class RecipeDetailPage extends React.Component {
         <QueryRecipe pk={recipeId} />
         <Row gutter={24}>
           <Col span={16}>
-            <DetailsActionBar />
             <RevisionNotice revision={revision} />
+            <Row type="flex" align="middle">
+              <Col span={4}>
+                <ShieldIdenticon seed={recipeId} letter={revision.getIn(['recipe', 'name'])} />
```

Comment:
If the `seed` value is just the `recipeId`, we can probably change `seed` to just `id` or something similar. The term `seed` was more appropriate when it was more directly responsible for the shield generation.

Reply:
Since this should be using the `identicon_seed` attribute from the recipe, I think that `seed` is appropriate.